CHANGELOG.md

```diff
@@ -1,21 +1,39 @@
 # App Center SDK for .NET Change Log
 
-## Version 4.0.1 (under development)
+## Version 4.1.0
 
 ### AppCenter
 
 * **[Feature]** Add a `SetMaxStorageSizeAsync` API which allows setting a maximum size limit on the local SQLite storage. The default value is 10MiB.
 
+#### iOS
+
+* **[Fix]** Fix a crash when SQLite returns zero for `page_size`.
+
 ### App Center Distribute
 
 #### iOS
 
 * **[Feature]** Add `WillExitApp` callback to distribute listener.
 
+#### Android
+
+* **[Fix]** Fix a crash when the app is trying to open the system settings screen from the background.
+* **[Fix]** Fix browser opening when using a private distribution group on Android 11.
+
 #### iOS/Android
 
  * **[Feature]** Add `NoReleaseAvailable` callback to distribute listener.
 
+ ### App Center Crashes
+
+ #### Android
+
+* **[Fix]** Fix removing throwable files after rewriting error logs due to small database size.
+
+#### iOS
+
+* **[Fix]** Fix nullability for attachments methods in crashes delegate. `nil` is valid return value now (4.0.0 regression).
```

Comment:
please move it to here https://github.com/microsoft/appcenter-sdk-dotnet/pull/1484/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15